endpoints.md

```diff
@@ -18,6 +18,8 @@
 ## Endpoints Under Development
 * /api/core/{model}/search <https://github.com/DSpace/DSpace/pull/1726>
 * [/api/discover/search](search-endpoint.md)
+* [/api/configuration/input-forms](input-forms.md)
+* [/api/configuration/submission-panels](submission-panels.md)
```

Comment:
Silly thing, but do we need to spell out "configuration" in these endpoints?  Why not shorten it to `/api/config/`?

Reply:
yes, I agree. We are going to apply the change